src/components/assets/Assets.vue

```diff
@@ -5,6 +5,7 @@
     </div>
     <div class="container--assets">
       <astar-button @click="unify()">Unify accounts</astar-button>
+      <astar-button @click="setIdentity()">Set setIdentity</astar-button>
```

Comment:
An extra 'set'

Reply:
Those are temporary buttons so I can test pieces of logic